### DISCUSSION TITLE
Add Buttons status to support A11y + Address common "disabled" inclusive language issue

### DISCUSSION THREAD

**Discussion by HfernetyA11y** [react-native-community/discussions-and-proposals#677] [Ideas]
Summary:
 Provide inactive button status of “unavailable” to improve accessibility for screen readers.     

Basic Example:
A transaction cannot be processed until require data is provided by casher.  Transaction “Pay” button needs to be in inactive status until data is provided.  A11y screen reader users need a way to hear that “pay” button is “unavailable”.   Motivations:
1-    Button UI element has no status support in the framework currently.
2-    While making this change consider also inclusive language enhancement.  Instead of using the common “disable” term use “unavailable”. 
 
Alternatives: 
The term “dimmed” could be used.  But we tested the term “unavailable” with lived experiences screen reader user and it was selected as it communicates more directly the status.

**Comment by PurpleMonkey729** [comment]
@HfernetyA11y !
I think the word "unavailable", "disable" and "dimmed" are all good.
What is the most important is to provide the new function and improve the accessibility for screen readers.
As a react native developer, I greet to your provision.
Best regards!
